cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -10,6 +10,9 @@
 from django.dispatch import receiver
 from pytz import UTC
 
+from edx_event_bus_kafka.publishing.event_producer import send_to_event_bus
```

Comment:
This is supposed to be configurable (ie we should be able to choose from which event bus library we're calling send_to_event_bus rather than assuming edx_event_bus_kafka).

Reply:
See https://github.com/edx/internal-dockerfiles/blob/master/edx-platform-private/Dockerfile for we added a dependency *only* for edx.org (the edx-arch-experiments line) for the containerized version of edx-platform. This may or may not be applicable for the current live version of edx-platform.

Reply:
For the moment we're likely to keep the explicit reference (under a try-import-except), pending https://github.com/openedx/openedx-events/pull/85

Reply:
See for latest: https://github.com/openedx/openedx-events/issues/88